### PR TITLE
fix: sass warnings

### PR DESCRIPTION
**Pull request by tomleo**
### 🔗 Linked issue

- https://energysage.atlassian.net/browse/CED-1731

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Upgrade dart-sass version
- Fix deprecation errors (`@debug` was super helpful)

### 🥼 Testing

- `npm run css-compile` in the `es-bs-base` folder/project

#### 🧐 Feedback Requested / Focus Areas

-

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.
- [ ] I have documented testing approach

### CODE REVIEW

**Comment by tomleo** [comment]
This removes the need for https://github.com/EnergySage/es-ds/pull/1471

File: es-bs-base/scss/_navbar.scss
Comment (nathanielwarner) [inline es-bs-base/scss/_navbar.scss:1]: It seems like this file was changed in a way that also makes the styling no longer equivalent, based on building the package and using within the old design system:

Screenshot 2024-08-14 at 16 12 26 (1)

**Review by nathanielwarner — CHANGES_REQUESTED** [review]
Following up from the Slack discussion we had, it seems like this is mostly good except for the navbar SASS changes.

**Comment by tomleo** [comment]
Closing in favor of https://github.com/EnergySage/es-ds/pull/1495

**Pull request closed without merging**